viite-UI/src/view/ProjectForm.js

```diff
@@ -6,6 +6,18 @@
     var formCommon = new FormCommon('');
     var ProjectStatus = LinkValues.ProjectStatus;
     var editableStatus = [ProjectStatus.Incomplete.value, ProjectStatus.Unknown.value];
+
+    // state to keep track if project links have been recalculated after changes made to project links
```

Comment:
Kielenhuoltoa:
"the project links", "the changes made", "to the project links"
Ehkä myös
"flag" parempi kuin "state", koska boolean. (Nillinnillinni, hth, hand <3)